src/cli/fuzzer_bridge/models.py

```diff
@@ -0,0 +1,140 @@
+
+    class Config:
+        """Pydantic configuration."""
+
+        populate_by_name = True
+
+
+class FuzzerEnvironment(BaseModel):
+    """Environment definition in fuzzer output."""
+
+    currentCoinbase: Address = Field(..., alias="currentCoinbase")
+    currentDifficulty: HexNumber = Field(HexNumber(0), alias="currentDifficulty")
+    currentGasLimit: HexNumber = Field(..., alias="currentGasLimit")
+    currentNumber: HexNumber = Field(..., alias="currentNumber")
+    currentTimestamp: HexNumber = Field(..., alias="currentTimestamp")
+    currentBaseFee: Optional[HexNumber] = Field(None, alias="currentBaseFee")
+    currentRandom: Optional[Hash] = Field(None, alias="currentRandom")
+    currentExcessBlobGas: Optional[HexNumber] = Field(None, alias="currentExcessBlobGas")
+    currentBlobGasUsed: Optional[HexNumber] = Field(None, alias="currentBlobGasUsed")
+    parentBeaconBlockRoot: Optional[Hash] = Field(None, alias="parentBeaconBlockRoot")
+    parentUncleHash: Optional[Hash] = Field(None, alias="parentUncleHash")
+    parentDifficulty: Optional[HexNumber] = Field(None, alias="parentDifficulty")
+    parentBaseFee: Optional[HexNumber] = Field(None, alias="parentBaseFee")
+    parentGasUsed: Optional[HexNumber] = Field(None, alias="parentGasUsed")
+    parentGasLimit: Optional[HexNumber] = Field(None, alias="parentGasLimit")
+
+    class Config:
+        """Pydantic configuration."""
+
+        populate_by_name = True
+
+
+class FuzzerOutput(BaseModel):
+    """Main fuzzer output format v2."""
+
+    version: str = Field(..., pattern="^2\\.0$")
+    fork: str
+    chainId: int = Field(1, alias="chainId")
+    accounts: Dict[Address, FuzzerAccount]
+    transactions: List[FuzzerTransaction]
+    env: FuzzerEnvironment
+    parentBeaconBlockRoot: Optional[Hash] = Field(None, alias="parentBeaconBlockRoot")
+
+    @field_validator("version")
+    @classmethod
+    def validate_version(cls, v):
+        """Ensure version is 2.0."""
+        if v != "2.0":
+            raise ValueError(f"Only version 2.0 is supported, got {v}")
+        return v
+
+    @field_validator("accounts", mode="before")
+    @classmethod
+    def validate_accounts(cls, v):
+        """Convert account addresses to Address type."""
+        if not v:
+            return {}
+        return {
+            Address(addr): (FuzzerAccount(**acc_data) if isinstance(acc_data, dict) else acc_data)
+            for addr, acc_data in v.items()
+        }
+
+    def validate_private_keys(self) -> None:
+        """Validate that all transaction senders have private keys."""
+        senders = {tx.from_ for tx in self.transactions}
+        for sender in senders:
+            if sender not in self.accounts:
+                raise ValueError(f"Sender {sender} not found in accounts")
+            if not self.accounts[sender].privateKey:
+                raise ValueError(f"No private key for sender {sender}")
```

Comment:
This can be converted to a `model_post_init` which pydantic can run automatically for verification:
```suggestion
    def model_post_init(self, __context):
        """Ensure all transactions have private keys."""
        super().model_post_init(__context)
        senders: Dict[Address, EOA] = {}
        for addr, account in self.accounts.items():
            if account.private_key is None:
                continue
            sender = EOA(key=account.private_key)
            assert Address(sender) == addr, (
                f"Private key for account {addr} does not match derived address {sender.address}"
            )
            senders[addr] = sender
        for tx in self.transactions:
            assert tx.from_ in senders, f"Sender {tx.from_} not found in accounts"
            tx.sender = senders[tx.from_]
```